modules/keepalived-limitation-for-ipi.adoc

```diff
@@ -0,0 +1,48 @@
+[discrete]
+[id="keepalived-limitation-for-ipi_{context}"]
+= Identifying pre-existing virtual routers
```

Comment:
Identifying virtual routers conflicts

@eslutsky
@janosdebugs
is that title makes sense?

Reply:
It makes sense, but I wouldn't be looking for this as a sysadmin. I would recommend rewording it, but I don't have a good grasp on what would make sense.

Reply:
Done.